### PR TITLE
Add instructions for transitioning from phase 1.0 to phase 1.1

### PR DESCRIPTION
**Pull request by asaj**
### Description
Instructions for restarting your validator after the network resets.

### Tested

To be tested on baklavastaging

### Other changes

Change network ID for baklava

### CODE REVIEW

File: packages/docs/getting-started/reconnecting-to-network.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+# On your validator machine
+cd celo-validator-node
+rm -rf geth* && rm static-nodes.json
+```
+
+## Restarting the components
+
+### Pulling the latest Celo Docker image
+
+First, pull the Celo image as described at the (Pull the Celo Docker image)[running-a-validator.md#pull-image] section. Be sure to pull the image on your local, validator, proxy, and attestations machines.
```
Comment (lucasege) [inline packages/docs/getting-started/reconnecting-to-network.md]: Should be `[Pull the Celo Docker image](running-a-validator.md#pull-image)`

File: packages/docs/getting-started/reconnecting-to-network.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+echo CELO_VALIDATOR_SIGNER_ADDRESS=$CELO_VALIDATOR_SIGNER_ADDRESS
+echo CELO_VALIDATOR_SIGNER_PUBLIC_KEY=$CELO_VALIDATOR_SIGNER_PUBLIC_KEY
+echo CELO_VALIDATOR_SIGNER_SIGNATURE=$CELO_VALIDATOR_SIGNER_SIGNATURE
+echo CELO_VALIDATOR_SIGNER_BLS_PUBLIC_KEY=$CELO_VALIDATOR_SIGNER_BLS_PUBLIC_KEY
+echo CELO_VALIDATOR_SIGNER_BLS_SIGNATURE=$CELO_VALIDATOR_SIGNER_BLS_SIGNATURE
+```
+
+If any of the environment variables are missing, you have two options:
+
+1.  Find the shell when you run the instructions and look for the environment variable missing. You can export it again using the following command:
```
Comment (lucasege) [inline packages/docs/getting-started/reconnecting-to-network.md]: Nit:
Replace
>  Find the shell when you run the instructions

With:
Find the shell **where you ran the original instructions**

**Review by lucasege — APPROVED** [review]

File: packages/docs/getting-started/running-a-validator.md [hunk 3]
```diff
@@ -116,11 +118,12 @@ First we are going to setup the main environment variables related with the `Bak
 
 ```bash
 export CELO_IMAGE=us.gcr.io/celo-testnet/celo-node:baklava
-export NETWORK_ID=12219
+export NETWORK_ID=76172
 ```
 
 ### Pull the Celo Docker image
 
+<a name="pull-image"></a>
```
Comment (marekolszewski) [inline packages/docs/getting-started/running-a-validator.md]: The name attribute on the <a> tag is not supported in HTML5. We should use a <span> tag with an id attribute instead.

File: scripts/run-docker-validator-network.sh [hunk 4]
```diff
@@ -270,7 +270,7 @@ if [[ $COMMAND == *"run-fullnode"* ]]; then
 
     echo -e "\tStarting the Full Node"
 
-    screen -S celo-fullnode -d -m docker run --name celo-fullnode --restart always -p 127.0.0.1:8545:8545 -p 127.0.0.1:8546:8546 -p 30303:30303 -p 30303:30303/udp -v $PWD/fullnode:/root/.celo $CELO_IMAGE --verbosity 3 --networkid $NETWORK_ID --syncmode full --rpc --rpcaddr 0.0.0.0 --rpcapi eth,net,web3,debug,admin,personal --lightserv 90 --lightpeers 1000 --maxpeers 1100 --etherbase $CELO_ACCOUNT_ADDRESS --ethstats=fullnode-$ETHSTATS_ARG
+    screen -S celo-fullnode -d -m docker run --name celo-fullnode --restart always -p 127.0.0.1:8545:8545 -p 127.0.0.1:8546:8546 -p 30303:30303 -p 30303:30303/udp -v $PWD/fullnode:/root/.celo $CELO_IMAGE --verbosity 3 --networkid $NETWORK_ID --syncmode full --rpc --rpcaddr 0.0.0.0 --rpcapi eth,net,web3,debug,admin,personal,txpool --lightserv 90 --lightpeers 1000 --maxpeers 1100 --etherbase $CELO_ACCOUNT_ADDRESS --ethstats=fullnode-$ETHSTATS_ARG
```
Comment (marekolszewski) [inline scripts/run-docker-validator-network.sh]: If this is running a validator, we should call the screen session: celo-validator.

File: scripts/run-docker-validator-network.sh [hunk 5]
```diff
@@ -270,7 +270,7 @@ if [[ $COMMAND == *"run-fullnode"* ]]; then
 
     echo -e "\tStarting the Full Node"
```
Comment (marekolszewski) [inline scripts/run-docker-validator-network.sh]: Change to: Starting the validator.

File: packages/docs/getting-started/reconnecting-to-network.md [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+
+## Double check that you still have your keys
+
+Make sure that you still have your Validator, Validator Group, and Validator signer private keys, as you will be able to re-use them in the next phase of the network. You can do so by listing the contents of your `keystore` directory on each machine. You should see a file ending with the address of the corresponding key.
+
+```bash
+# On your local machine
+# You should see the keystore file for your Validator and Group keys.
+ls celo-accounts-node/keystore
+```
```
Comment (clrblmt) [inline packages/docs/getting-started/reconnecting-to-network.md:35]: Add what output should look like, or mention should see the `keystore` folder

File: packages/docs/getting-started/reconnecting-to-network.md [hunk 7]
```diff
@@ -0,0 +1,148 @@
...
+
+```bash
+# On your validator machine
+# You should see the keystore file for your Validator signer key.
+ls celo-validator-node/keystore
+```
```
Comment (clrblmt) [inline packages/docs/getting-started/reconnecting-to-network.md:41]: Same comment as above

File: packages/docs/getting-started/reconnecting-to-network.md [hunk 8]
```diff
@@ -0,0 +1,148 @@
+# How to reconnect to Baklava after a network reset
+
+If you were running a Validator and Proxy in a previous phase of the `baklava` network, this page contains instructions about how to set up your Validator on the current phase.
```
Comment (marekolszewski) [inline packages/docs/getting-started/reconnecting-to-network.md:3]: @asaj Should we add an extra sentence here that says: "If you've automated running a Celo validator with a script, please review these and the "Running a Validator" instructions again as there have been a number of small updates and security-related changes."

**Pull request merged**